### ISSUE TITLE
Make add_pb_suffix the default behaviour

### ISSUE THREAD

**Issue by timostamm** [timostamm/protobuf-ts#199]
https://github.com/timostamm/protobuf-ts/commit/5e0792557a5614cb0394a9c895c39d6edca509f7 added the plugin option:

> - "add_pb_suffix"
  Adds the suffix _pb to the names of all generated files. This will become the 
  default behaviour in the next major release.

This should be made the default behaviour in v3.0 so that users don't have to reason about it. A plugin option `dont_add_pb_suffix` (maybe with a better name) should be added to revert to the old behaviour to ease migration.

**Comment by jcready** [comment]
All the currently supported CLI options for the `protoc` plugin are simply flags (booleans where their presence means `true` and absence means `false`). I _believe_ it should be possible to add support for proper CLI options such that we don't have to add/remove conflicting (mutually exclusive) options between major versions.

Something like: `add_pb_suffix=false` or `filename_suffix='_pb'`. I can think of several other existing mutually exclusive flags that might benefit from this: `long_type=string|bigint|number`, `optimize=size|speed`, etc.

**Comment by timostamm** [comment]
I agree that it's about time for value-options. `long_type`, `client`, `server` and `optimize` are all great candidates for it, and `output_javascript` added by https://github.com/timostamm/protobuf-ts/pull/200 as well. But I'm not sure it will help with breaking changes.

Whether the `_pb` suffix is worth a breaking change is another question. On the one hand, it would be annoying for users with a large existing code base, on the other hand it's worth maintaining sensible defaults. So far, https://github.com/timostamm/protobuf-ts/issues/143 is one the road map for v3. I think it should be fair to point to the two breaking changes in the release notes, and explain what needs to be changed for the update?
